client/src/backend/containers/records/export-targets/List.js

```diff
@@ -0,0 +1,93 @@
+import React, { PureComponent } from "react";
+import PropTypes from "prop-types";
+import connectAndFetch from "utils/connectAndFetch";
+import { entityStoreActions } from "actions";
+import { select, meta } from "utils/entityUtils";
+import { exportTargetsAPI, requests } from "api";
+import get from "lodash/get";
+import lh from "helpers/linkHandler";
+import { childRoutes } from "helpers/router";
+import EntitiesList, {
+  Button,
+  ExportTargetRow
+} from "backend/components/list/EntitiesList";
+
+const { request } = entityStoreActions;
+
+export class ExportTargetsContainerImplementation extends PureComponent {
+  static mapStateToProps = state => {
+    return {
+      exportTargets: select(requests.beExportTargets, state.entityStore),
+      exportTargetMeta: meta(requests.beExportTargets, state.entityStore)
+    };
+  };
+
+  static displayName = "ExportTargets.List";
+
+  static propTypes = {
+    exportTargets: PropTypes.array,
+    exportTargetsMeta: PropTypes.object,
+    match: PropTypes.object.isRequired,
+    route: PropTypes.object.isRequired,
+    dispatch: PropTypes.func.isRequired
+  };
+
+  static defaultProps = {
+    exportTargets: [],
```

Comment:
I'd have it be null. Empty array is a confusing state for not having received data from the server.

Reply:
Is it better in this case to set the `defaultProp` value on an `exportTarget` to null, or just to not have a `defaultProp` value at all?

Reply:
Better not to have the default prop value at all if it's null.